expo/withAppsFlyerAndroid.js

```diff
@@ -0,0 +1,32 @@
+const {withMainActivity} = require('@expo/config-plugins')
+
+function overrideOnNewIntent(contents, packageName = ''){
+    let nextContent = contents
+    const intentImportString = 'import android.content.Intent'
+
+    if (!nextContent.includes(intentImportString)){
+        const packageString = `${packageName}\n`
+        nextContent = nextContent.replace(packageString,`${packageString}\n${intentImportString}`)
```

Comment:
Add a new line at the end of the import.
```suggestion
        nextContent = nextContent.replace(packageString,`${packageString}\n${intentImportString}\n`)
```